### PR TITLE
Translate template.html to English

### PR DESCRIPTION
**Pull request by JacquesTa**
fixes https://github.com/serlo/overview.serlo.org/issues/68

### CODE REVIEW

File: template.html [hunk 1]
```diff
@@ -117,9 +117,9 @@ <h1 id="{{name|urlencode}}">{{name}}</h1>
             {% macro toc_item(name) %}
             <li class="nav-item"><a class="nav-link" href="#{{name|urlencode}}">{{name}}</a></li>
             {% endmacro %}
-            {{ toc_item("Projekte") }}
+            {{ toc_item("Project") }}
```
Comment (kulla) [inline template.html:120]:
```suggestion
            {{ toc_item("Projects") }}
```

File: template.html [hunk 2]
```diff
@@ -117,9 +117,9 @@ <h1 id="{{name|urlencode}}">{{name}}</h1>
...
             {{ toc_item("Support-Units") }}
-            {{ toc_item("Teammitglieder") }}
+            {{ toc_item("Team member") }}
```
Comment (kulla) [inline template.html:122]:
```suggestion
            {{ toc_item("Team members") }}
```

File: template.html [hunk 3]
```diff
@@ -168,9 +168,9 @@ <h1 id="{{name|urlencode}}">{{name}}</h1>
           {{ render_units(serlo.projects, "Projekte") }}
           {{ render_units(serlo.support_units, "Support-Units") }}
 
-          {{ section("Teammitglieder") }}
+          {{ section("Team member") }}
```
Comment (kulla) [inline template.html:171]:
```suggestion
          {{ section("Team members") }}
```

File: template.html [hunk 4]
```diff
@@ -168,9 +168,9 @@ <h1 id="{{name|urlencode}}">{{name}}</h1>
...
           {% call(member) generate_table(serlo.persons,
-            ["Name", "Kontakt", "Projekte / Units", "Verantwortung"]) %}
+            ["Name", "Contact", "Project / Units", "Responsability"]) %}
```
Comment (kulla) [inline template.html:173]:
```suggestion
            ["Name", "Contact", "Projects / Units", "Sherpa support"]) %}
```

File: template.html [hunk 5]
```diff
@@ -168,9 +168,9 @@ <h1 id="{{name|urlencode}}">{{name}}</h1>
           {{ render_units(serlo.projects, "Projekte") }}
```
Comment (kulla) [inline template.html:168]:
```suggestion
          {{ render_units(serlo.projects, "Projects") }}
```

File: template.html [hunk 6]
```diff
@@ -144,7 +144,7 @@ <h1 id="{{name|urlencode}}">{{name}}</h1>
           {% macro render_units(units, name) %}
             {{ section(name) }}
             {% call(project) generate_table(units|sort(attribute="title"),
-              ["Name", "Lama", "Mitglieder", "Beschreibung"]) %}
+              ["Name", "Lama", "Member", "Description"]) %}
```
Comment (kulla) [inline template.html:147]:
```suggestion
              ["Name", "Lama", "Members", "Description"]) %}
```

**Comment by JacquesTa** [comment]
@kulla ready to be merge

**Pull request merged**